src/lib/protocols/v3/export.ts

```diff
@@ -0,0 +1,64 @@
+import { deflate } from 'pako'
+import { base64url } from 'rfc4648'
+
+import type { ProtocolVersion, ReflectionEntry } from '$lib/types'
+import { encodeInt32, encodeString, mergeTypedArrays } from '$lib/utils'
+import { PROTOCOL_VERSION } from './protocol'
+
+function encodeTime(date: Date) {
+    const timestamp = date.getTime() / 1000
+    return encodeInt32(timestamp)
+}
+
+function encodeEntry(entry: ReflectionEntry) {
+    return mergeTypedArrays(
+        encodeTime(entry.time),
+        new Uint8Array(encodeEntryData(entry.data)),
+        encodeInt32(entry.comment != null ? entry.comment.length : 0),
+        encodeString(entry.comment != null ? entry.comment : 'null'))
```

Comment:
Should we set the comment to an empty string instead? If we set a string with the value `'null'`, this is 4 bytes which wouldn't match the expected string length, and so other entries might get corrupted since the next entry doesn't start unil 4 characters later.

We probably want an empty string here instead, or what do you think?

```suggestion
        encodeString(entry.comment != null ? entry.comment : ''))
```